scripts/ensure-python.sh

```diff
@@ -19,7 +19,7 @@ fi
 # Warn about build deps.
 echo "Warning: You will likely need to install build dependencies for your system."
 echo "e.g. Ubuntu requires: build-essential libreadline-dev tk-dev tcl-dev"
-read -pr "Continue? [Y/n] " ans
+read -r -p "Continue? [Y/n] " ans
```

Comment:
Good catch. I just used this script a few days ago--not sure how it didn't error on me. I may have modified it after having used it. The "message" is an option argument to `-p`, so it has to follow directly behind it. There's no way this should have worked before, regardless of bash version.